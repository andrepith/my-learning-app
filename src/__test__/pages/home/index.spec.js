import ShallowRenderer from 'react-test-renderer/shallow';
import Home from '../../../pages/home';

it('Renders home to match snapshot', () => {
  const renderer = new ShallowRenderer();
  const tree = renderer.render(<Home />);
  expect(tree).toMatchSnapshot();
});
