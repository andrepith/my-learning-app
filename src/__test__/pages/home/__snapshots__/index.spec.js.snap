// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders home to match snapshot 1`] = `
<div
  className="home"
>
  <h1>
    My Learning App
  </h1>
  <div
    className="available-class"
  />
</div>
`;
